parser/src/ast/integrity_constraints.rs

```diff
@@ -1,11 +1,16 @@
-use super::{EvaluatorFunctionCall, Expression, Variable};
+use super::{EvaluatorFunctionCall, Expression, Identifier, Iterable, Variable};
 
 // INTEGRITY STATEMENTS
 // ================================================================================================
 
 #[derive(Debug, Eq, PartialEq)]
 pub enum IntegrityStmt {
     Constraint(ConstraintType, Option<Expression>),
+    ConstraintComprehension(
+        ConstraintType,
+        Option<Expression>,
+        ConstraintComprehensionContext,
+    ),
```

Comment:
I believe the only difference between this and the one above is the `ConstraintcomprehensionContext`, so I would probably prefer to take the same approach we took for selectors and do the following:

```
pub enum IntegrityStmt {
    Constraint(ConstraintType, Option<ConstraintComprehensionContext>, Option<Expression>),
    Variable(Variable),
}
```

I would also prefer to put the constraint comprehension context before the selector expression, since I think this is the way the language syntax should be ordered, as I noted elsewhere